project.yaml

```diff
@@ -1,12 +1,25 @@
-version: '3.0'
+version: "3.0"
 
 expectations:
   population_size: 1000
 
 actions:
-
-  generate_study_population:
-    run: cohortextractor:latest generate_cohort --study-definition study_definition
+  generate_dataset:
+    run: >
+      databuilder:v0.34.1
+        generate_dataset
+        --dataset-definition analysis/dataset_definition.py
+        --dataset output/dataset.csv
+        --dummy-data-file dummy_output/dataset.csv
```

Comment:
Can `generate_dataset` generate, for example, `.csv.gz` files?

Reply:
I don't think so, we're not importing `tarfile` anywhere so I'd assume not for now.